SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -266,9 +267,20 @@ void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FStr
 	FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
 	TWeakPtr<SSpatialOutputLog> SpatialOutputLog = GDKServices.GetSpatialOutputLog();
 
-	RuntimeProcess->OnOutput().BindLambda([&, SpatialOutputLog](const FString& Output) {
+	RuntimeProcess->OnOutput().BindLambda([&, SpatialOutputLog](FString& Output) {
+		// Format and output the log to the editor window `SpatialOutputLog`
 		SpatialOutputLog.Pin()->FormatAndPrintRawLogLine(Output);
 
+		// Save the raw runtime output to disk.
+		if (RuntimeLogFileHandle)
```

Comment:
Explicit pointer nullptr check maybe?

Reply:
is that house style no relying on implicit bool conversions for if statements?

Reply:
I think it is unreals style

Reply:
Converting to `TUniquePtr` so using `IsValid()` now.